mmcv/cnn/bricks/activation.py

```diff
@@ -1,12 +1,14 @@
 import torch
 import torch.nn as nn
+import torch.nn.functional as F
 
-from mmcv.utils import build_from_cfg
+from mmcv.utils import TORCH_VERSION, build_from_cfg
 from .registry import ACTIVATION_LAYERS
 
 for module in [
         nn.ReLU, nn.LeakyReLU, nn.PReLU, nn.RReLU, nn.ReLU6, nn.ELU,
-        nn.Sigmoid, nn.Tanh
+        nn.Sigmoid, nn.Tanh, F.gelu
+        if TORCH_VERSION == 'parrots' or TORCH_VERSION <= '1.3.1' else nn.GELU
 ]:
     ACTIVATION_LAYERS.register_module(module=module)
```

Comment:
I suggest wrapping a GELU module for PyTorch 1.3.x since `F.gelu` cannot be called like `nn.GELU`.

Reply:
Got it! Thanks a lot!